public/js/date.js

```diff
@@ -0,0 +1,47 @@
+const tDate = (
+
+  function tStamp(dateRec) {
+
+    var timeStamp = '';
```

Comment:
Should not use `var` anywhere in your JS code. Use rule either `const` or `let`.

A general rule to follow:
Always use `const` UNLESS you plan on reassigning the value, in which case you should use `let`